### PR TITLE
Release for 2.12

### LINKED ISSUE #45: 2.12 release
I'd guess you'd need a 2.12 version of https://github.com/Verizon/sbt-rig beforehand, I'm hitting

    [warn]  Note: Unresolved dependencies path:
    [warn]          org.scalatest:scalatest_2.12:2.2.6 ((verizon.build.common) RigPlugin.scala#L79)
    [warn]            +- io.verizon.knobs:knobs_2.12:3.11.0-SNAPSHOT
    [warn]          org.scalacheck:scalacheck_2.12:1.12.5 ((verizon.build.common) RigPlugin.scala#L79)
    [warn]            +- io.verizon.knobs:knobs_2.12:3.11.0-SNAPSHOT
    [warn]          org.scoverage:scalac-scoverage-runtime_2.12:1.1.1 ((scoverage.ScoverageSbtPlugin) ScoverageSbtPlugin.scala#L30)
    [warn]            +- io.verizon.knobs:knobs_2.12:3.11.0-SNAPSHOT
    [warn]          org.scoverage:scalac-scoverage-plugin_2.12:1.1.1 ((scoverage.ScoverageSbtPlugin) ScoverageSbtPlugin.scala#L30)
    [warn]            +- io.verizon.knobs:knobs_2.12:3.11.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by rossabaker**
- Introduces builds for Scala 2.12 (supersedes #47)
- Introduces `ValueError` (includes #43)
- Bumps knobs version to v3.13
- Published with `-scalaz-7.x` version qualifier.  This will be Verizon's convention moving forward.

Fixes #45

### CODE REVIEW

**Comment by rossabaker** [comment]
This depends on a release of Verizon/ermine-parser#3.

File: core/src/main/scala/knobs/Trivial.scala [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package knobs
```
Comment (timperrett) [inline core/src/main/scala/knobs/Trivial.scala]: @rossabaker File needs a copyright notice.

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-version in ThisBuild := "3.12.0-SNAPSHOT"
+version in ThisBuild := "3.13.0-SNAPSHOT"
```
Comment (timperrett) [inline version.sbt]: Given the changes here, should we do a major revision? This seems like we introduced some changes we know are not compatible. WDYT?

File: project/ScalazPlugin.scala [hunk 3]
```diff
@@ -0,0 +1,64 @@
+package verizon.build
```
Comment (timperrett) [inline project/ScalazPlugin.scala]: File needs a copyright notice.

File: version.sbt (hunk 2 above)
Comment (timperrett) [inline version.sbt]: Actually here's a more compelling reason: we're dropping support for scalaz-stream 0.7

Comment (ppurang) [inline version.sbt]: +1 to major version 4

**Pull request merged**